docs/how-to-enable-new-languages.md

```diff
@@ -281,3 +281,130 @@ Once these are in place, you should be able to run `npm run develop` to view you
 
 > [!ATTENTION]
 > While you may perform translations locally for the purpose of testing, we remind everyone that translations should _not_ be submitted through GitHub and should only be done through Crowdin. Be sure to reset your local codebase after you are done testing.
+
+# Deploying New Languages on `/news`
+
+To deploy News for a new language, you'll need to create two PRs. One PR will be to the [CDN repo](https://github.com/freeCodeCamp/cdn), and the other will be to the [News repo](https://github.com/freeCodeCamp/news).
+
+## Prep the CDN Repo for the New Language
+
+News sources trending links and article titles from our CDN during the build and adds them to the footer. News also fetches dayjs files from the CDN during the build to localize dates and times for each language.
+
+### Add a YAML File for Trending Articles
+
+Clone the [CDN repo](https://github.com/freeCodeCamp/cdn) and create a new branch.
+
+In the [`build/universal/trending`](https://github.com/freeCodeCamp/cdn/tree/main/build/universal/trending) directory, create a new file and name it `language.yaml`. For example, if you are launching Dothraki News, name the file `dothraki.yaml`.
+
+Then copy the contents of the [`english.yaml`](https://github.com/freeCodeCamp/cdn/blob/main/build/universal/trending/english.yaml) trending file and paste it into the new YAML file you just created.
+
+The contents will look something like this:
+
+```yaml
+article0title: 'Learn JavaScript'
+article0link: 'https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/'
+article1title: 'Linux ln Example'
+article1link: 'https://www.freecodecamp.org/news/linux-ln-how-to-create-a-symbolic-link-in-linux-example-bash-command'
+article2title: 'JS document.ready()'
+article2link: 'https://www.freecodecamp.org/news/javascript-document-ready-jquery-example/'
+article3title: ...
+article3link: ...
+  ...
+```
+
+### Add a dayjs file for the New Language
+
+In the [`build/news-assets/dayjs/1.10.4/locale`](https://github.com/freeCodeCamp/cdn/tree/main/build/news-assets/dayjs/1.10.4/locale) directory, create a new file and name it `isocode.min.js`. For example, if you are launching Dothraki News, name the file `mis.min.js`.
+
+Then, visit [this page on cdnjs](https://cdnjs.com/libraries/dayjs/1.10.4) with all available dayjs v1.10.4 files, find the `https://cdnjs.cloudflare.com/ajax/libs/dayjs/1.10.4/locale/isocode.min.js` link for the new language, and open it in a new tab.
+
+Copy the dayjs code from the new tab into the new file you created. The code will be similar to this, but minified:
```

Comment:
I am sure we are talking about the `dayjs.min.js` file here. I believe it would be better to add the complete file name instead of `daysjs`. WYA'T?

Reply:
Hey @RafaelDavisH, sorry again if this was unclear.

I left a comment and a link to the file we need to add in your PR to the CDN repo: https://github.com/freeCodeCamp/cdn/pull/163#issuecomment-1280433546

Also, I tried to rewrite the Day.js bits so it's more clear that the only thing we need to add is a Day.js locale file for the new language.

Reply:
Sounds good @scissorsneedfoodtoo. I made the change to days.min.js file and push it to the PR #163